libs/stats/README.md

```diff
@@ -13,21 +12,176 @@ pip install --extra-index-url="https://packages.dea.ga.gov.au" odc-stats
 
 Usage
 -----
+Stats offers a set of tools to generate clear pixel count and geometric median on Sentinel-2 and Landsat satellite images and it's currently being extended to support sibling products.
 
-### Prepare tasks for processing:
+### Steps to run stats
 
-From a machine that has access to your database, run:
+### 1- Save tasks
+
+From your sandbox (or a machine that has access to your database), run:
+
+```
+odc-stats save-tasks --frequency annual --temporal-range 2015--P6Y --grid au-10  ga_ls8c_ard_3
+```
+
+The above command will generate the following files i.e. a csv file containing a list of tasks for years between 2015 to 2020, a cache file that will be used as an input to the stats cli, and several geojson files (one for each year).  
+
+```
+ga_ls8c_ard_3_2015--P6Y-2015--P1Y.geojson  
+ga_ls8c_ard_3_2015--P6Y-2016--P1Y.geojson  
+ga_ls8c_ard_3_2015--P6Y-2017--P1Y.geojson  
+ga_ls8c_ard_3_2015--P6Y-2018--P1Y.geojson
+ga_ls8c_ard_3_2015--P6Y-2019--P1Y.geojson
+ga_ls8c_ard_3_2015--P6Y-2020--P1Y.geojson  
+ga_ls8c_ard_3_2015--P6Y.db
+ga_ls8c_ard_3_2015--P6Y.csv
+```
+
+The csv file contains the list of tasks for all the years and consists of x, y coordinates of each tile to be processed as well as the counts of datasets and satellite observations.
+
+geojson files are useful when selecting test regions as well as for debugging specific tasks - see the example below from Landsat-8, 2015.  This example shows a tile from Australia coastal region near Sydney.
+
+<img src="odc/stats/stats/auxiliary/screenshot-L8-2015.png" alt="drawing" width="1000"/>
+
+
+### 2- Run stats
+Sample command:
+```
+odc-stats run s3-path/ga_ls8c_nbart_gm_cyear_3.db  2015--P1Y/41/13 --threads=16 --memory-limit=60Gi --resolution=30 --config cfg.txt --location file://localpath/
```

Comment:
Let's get rid of `s3-path/` prefix, and make sure that db name is the same as was produced by the save-tasks step.

Reply:
Addressed accordingly.